DECISION-MAKING.md

```diff
@@ -0,0 +1,86 @@
+# Decision-making rules
+
+## Introduction
+
+The Brain Imaging Data Structure (BIDS) community set out the following 
+decision-making rules with the intention to:
+
+- Strive for consensus.
+- Promote open discussions.
+- Minimize the administrative burden.
+- Provide a path for when consensus cannot be made.
+- Grow the community.
+- Maximize the [bus factor](https://en.wikipedia.org/wiki/Bus_factor) of the 
+  project.
+
+The rules outlined below are inspired by the [lazy consensus system used in the Apache Foundation](https://www.apache.org/foundation/voting.html) 
+and heavily depends on [GitHub Pull Request Review system](https://help.github.com/articles/about-pull-requests/).
+
+## Definitions
+
+**Repository** - [https://github.com/bids-standard/bids-specification](https://github.com/bids-standard/bids-specification)
+
+**Contributor** - a person listed in the Appendix I: Contributors. The 
+community decides on the content of this file using the same process as any 
+other change to the Repository (see below) allowing the meaning of "Contributor"
+to evolve independently of the Decision-making rules.
+
+## Rules
+
+1. Every modification of the specification (including a correction of a typo, 
+   adding a new Contributor, an extension adding support for a new data type, or
+   others) or proposal to release a new version needs to be done via a Pull 
+   Request (PR) to the Repository.
+1. Anyone can open a PR (this action is not limited to Contributors).
+1. PRs adding new Contributors must also add their GitHub names to the 
+   [CODEOWNERS](CODEOWNERS) file.
```

Comment:
I don't quite follow this bullet point.

Is the idea that if you're opening a pull request to a file you should make sure you're notified about future changes to that file?

Would there be a way to remove yourself from CODEOWNERS? Or once someone has made an edit to a file, they'll always be watching it?

Reply:
@choldgraf requested this addition in https://github.com/bids-standard/bids-specification/pull/104#issuecomment-450186641 perhaps he can comment.

Reply:
My main goal was to prevent "maintainers" from becoming an ever-expanding list of people with no explicit obligations attached to being a maintainer. It seemed like "I agree to get notifications about changes to this repository" is a reasonable ask for people who want to officially be named as "maintainers".

Reply:
Quick clarification - we don't use the word "maintainer" anywhere in the spec.  The term that was used so far was "contributor".